otherlibs/runtime_events/runtime_events.mli

```diff
@@ -46,19 +46,66 @@
     very short running programs.
 *)
 
-(** The type for counter events emitted by the runtime. *)
+(** The type for counter events emitted by the runtime. Counter events are used
+  to measure a quantity at a point in time or record the occurance of an event.
+  In the latter case their value will be one. *)
 type runtime_counter =
 | EV_C_FORCE_MINOR_ALLOC_SMALL
+(**
+Triggering of a minor collection due to a full minor heap.
+@since 5.0
+*)
 | EV_C_FORCE_MINOR_MAKE_VECT
+(**
+Triggering of a minor collection due to Array.make.
+@since 5.0
+*)
 | EV_C_FORCE_MINOR_SET_MINOR_HEAP_SIZE
+(**
+Triggering of a minor collection due to Gc.minor_heap_size.
+@since 5.0
+*)
 | EV_C_FORCE_MINOR_MEMPROF
+(**
+Triggering of a minor collection during memprof young sampling.
+@since 5.3
+*)
 | EV_C_MINOR_PROMOTED
+(**
+Total words promoted from the minor heap to the major in the last minor
+collection.
+@since 5.0
+*)
 | EV_C_MINOR_ALLOCATED
+(**
+Total {b bytes} allocated in the minor heap in the last minor collection.
+@since 5.0
+*)
 | EV_C_REQUEST_MAJOR_ALLOC_SHR
+(**
+Major slice requested due to allocation in major heap.
+@since 5.0
+*)
 | EV_C_REQUEST_MAJOR_ADJUST_GC_SPEED
+(**
+Major slice requested due to deallocated resources during finalization.
+@since 5.0
+*)
```

Comment:
The comment for `EV_C_REQUEST_MAJOR_ADJUST_GC_SPEED` https://github.com/ocaml/ocaml/pull/13382/files#diff-be70d1226dd3c5eb34dec207e3697531d686927f55e970786b2504da9869cd7fR90-R93 makes sense when read alongside the code comment for `caml_adjust_gc_speed`  https://github.com/ocaml/ocaml/blob/8bbb37491457420ed393cb8d8d76bb59938934b2/runtime/memory.c#L256-L270 but not as much by itself. Maybe a rewording to mention the use of finalized blocks to automatically deallocate resources would improve it?

Reply:
Alternatively, how about 

> Major slice requested by [caml_adjust_gc_speed].

`caml_adjust_gc_speed` is an exported function, and hence, mentioning the function by name should be sufficient rather than duplicating the details in multiple places.

Reply:
Done